setup.py

```diff
@@ -10,18 +10,5 @@
     packages=find_packages(),
     version=__version__,
     entry_points={"console_scripts": "grants_tagger=grants_tagger.__main__:app"},
-    install_requires=[
```

Comment:
we still want this to act as a library but we want two set of requirements one for inference which should go here and an extra `dev` for the rest